pkg/module/go_get_fetcher.go

```diff
@@ -120,6 +115,38 @@ func getSources(goBinaryName string, fs afero.Fs, gopath, repoRoot, module, vers
 	return nil
 }
 
+func prepareEnv(gopath string) []string {
+	names := []string{
+		"GOPATH",
+		"GOCACHE",
+		"CGO_ENABLED",
+		"GO111MODULE",
+		"GOPROXY",
+	}
+	parentEnv := os.Environ()
+	shouldRemove := func(parentEnvVar string) bool {
```

Comment:
can you name it shouldKeep or something like that and switch the logic down there?